website/content/en/docs/deployment/cognito/guide-automated.md

```diff
@@ -71,14 +71,20 @@ This guide assumes you have Python 3.8 installed and that you have completed the
                     name: platform.example.com
                     us-east-1-certARN: arn:aws:acm:us-east-1:123456789012:certificate/373cc726-f525-4bc7-b7bf-d1d7b641c238
             ```
+1. Export the current deployment option
```

Comment:
lets move the DEPLOYMENT_OPTION export also into the tabpane so user can copy+paste in one shot. Duplicate content here but on the website it will appear as option

Same for other guides